core/src/main/java/com/oath/oak/MemoryManager.java

```diff
@@ -45,7 +47,18 @@ public ByteBuffer allocateKeys(int bytes) {
         return keysMemoryAllocator.allocate(bytes);
     }
 
-    public void releaseKeys(ByteBuffer keys) {
-        keysMemoryAllocator.free(keys);
+    public OakNativeMemoryAllocator.Slice allocateSlice(int bytes) {
```

Comment:
Can you please add a comment explaining when oak uses allocate slice vs allocate keys or allocate

Reply:
Allocate keys is no longer in use and I removed it.
Other than that the comment was added